### DISCUSSION TITLE
[Looking For] GoPhish Developers

### DISCUSSION THREAD

**Discussion by websecnl** [gophish/gophish#3036] [General]
Hi,

My name is Joel, I am the managing director at WebSec (A Dutch Cybersecurity company).
We are looking for developers who can implement new features into GoPhish such as

- Supporting multiple TLS certificates for multiple domains
- Integrating ZeroSSL API to create on-the-fly TLS/SSL certificates 1-2 days before Scheduled Campagne launch
- Ability to add custom tags such as {{.CustomTag1}} {{.CustomTag2}}
- Multiple new different types of variable names as alternative to the GET ?rid=9017 variable
example:  .com/tenant/9017/   or custom variable name, example:  .php?tenant_id=9017

And many more features, of which most are already under the 'Issues' tab.

We are willing to pay for the implementation of these features in order to have an improved version of GoPhish.

If you are interested then please send us your proposal to jobs@websec.nl

Note: this is a freelance position, you will be paid per requested feature.

Best Regards,
Joel Aviad Ossi

**Comment by EDMPhoenix** [comment]
Dear Mr. Joel Aviad Ossi,

I am writing to express my profound interest in your proposal for implementing new features into GoPhish as described under the 'Issues' tab. As an experienced software developer, I am confident in my capability to undertake this project.

Understanding the importance of robust security systems such as GoPhish, I have always been passionate about Cybersecurity and have demonstrated this in every task I have undertaken. I have a solid background in implementing and tailoring solutions that require intimate knowledge of TLS/SSL certificates, APIs, and custom tags, which makes working on your project a prospect that truly excites me.

Thank you for considering my proposal. I'll be happy to provide further information upon request. You can also direct any questions or further discussions about my proposal to this email.

I'm looking forward to potentially working together to enhance the productivity and efficacy of GoPhish.

Kindest Regards.

**Comment by websecnl** [comment]
What is your email address, we will send you a message.

**Reply by EDMPhoenix** [reply]
Hello @websecnl 

Thank you for reaching out to me.

Here is my gmail address. greatm1257@gmail.com

Best.
